modules/utxo-lib/test/integration_local_rpc/generate/outputScripts.util.ts

```diff
@@ -93,10 +97,15 @@ export function createScriptPubKey(keys: KeyTriple, scriptType: ScriptType, netw
   }
 }
 
+export type TxbInput = PrevOutput & {
```

Comment:
Kinda confusing naming here when `TxbInput` has a different type inside `TransactionBuilder`. Maybe `TxInput` would be more appropriate actually?

Reply:
I didn't realize these were different but I agree we should name them differently if they're not referring to the same type.

Reply:
well `TxInput` also exists and is even exported